From ed49f35a84a7eeafd273ed88fea3911bed58d24f Mon Sep 17 00:00:00 2001
From: Nicolas Le Bayon <nicolas.le.bayon@foss.st.com>
Date: Fri, 22 Nov 2024 14:44:51 +0100
Subject: [PATCH 741/954] fix(stm32mp21-fdts): new LPDDR4 LP4WL setting for
 STM32MP215F-DK board

Fix LP4WL formula in DDRCTRL_calc sheet.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git a52f607887f21455f6d1d97440dc64b36ba142f1 fix(stm32mp21-fdts): new LPDDR4 LP4WL setting for STM32MP215F-DK board]
Signed-off-by: Nicolas Le Bayon <nicolas.le.bayon@foss.st.com>
Change-Id: Ia36e7e53d58a6bb1537974f001b5c8b7acae575e
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/420374
Tested-by: Nicolas LE BAYON <nicolas.le.bayon@st.com>
Reviewed-by: Nicolas LE BAYON <nicolas.le.bayon@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 fdts/stm32mp21-lpddr4-1x16Gbits-1x16bits-800MHz.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fdts/stm32mp21-lpddr4-1x16Gbits-1x16bits-800MHz.dtsi b/fdts/stm32mp21-lpddr4-1x16Gbits-1x16bits-800MHz.dtsi
index 3358d3c903..8f1337e5f3 100644
--- a/fdts/stm32mp21-lpddr4-1x16Gbits-1x16bits-800MHz.dtsi
+++ b/fdts/stm32mp21-lpddr4-1x16Gbits-1x16bits-800MHz.dtsi
@@ -165,7 +165,7 @@
 #define DDR_UIA_CALINTERVAL 0x00000009
 #define DDR_UIA_CALONCE 0x00000000
 #define DDR_UIA_LP4RL_0 0x00000002
-#define DDR_UIA_LP4WL_0 0x00000003
+#define DDR_UIA_LP4WL_0 0x00000002
 #define DDR_UIA_LP4WLS_0 0x00000000
 #define DDR_UIA_LP4DBIRD_0 0x00000000
 #define DDR_UIA_LP4DBIWR_0 0x00000000
-- 
2.53.0

